src/VisualStudio/Core/Test.Next/UnifiedSettings/TestModel/UnifiedSettingBase.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Linq;
+using System.Text.Json.Serialization;
+using Roslyn.Utilities;
+
+namespace Roslyn.VisualStudio.Next.UnitTests.UnifiedSettings.TestModel;
+
+internal abstract record UnifiedSettingBase
```

Comment:
All these classes are just mapping to the `csharpSettings.registration.json` so it's easier to make assertions to verify the properties.